portfolio/src/test/javascript/bookListControllerTest.js

```diff
@@ -0,0 +1,23 @@
+describe('The book list controller ', function() {
```

Comment:
Let's also remove this code from this PR since we don't have it all working, yet.